File: src/main/java/com/github/knextsunj/cms/dto/CityDTO.java
package com.github.knextsunj.cms.dto;

import com.fasterxml.jackson.annotation.JsonProperty;

import java.io.Serializable;

public record CityDTO(@JsonProperty("id") Long id, @JsonProperty("name") String name,
                      @JsonProperty("deleted") String deleted,
                      @JsonProperty("stateId") Long stateId, @JsonProperty("stateName") String stateName) implements Serializable {
}

File: src/main/kotlin/com/github/knextsunj/cms/dto/CityDTO.kt
package com.github.knextsunj.cms.dto

import com.fasterxml.jackson.annotation.JsonProperty

data class CityDTO(@JsonProperty("id") val id:Long?, @JsonProperty("name") val name:String?,
                   @JsonProperty("deleted") val deleted:String?,
                   @JsonProperty("stateId") val stateId:Long?, @JsonProperty("stateName") val stateName:String?)
